src/event/event_kevent.c

```diff
@@ -1129,7 +1152,7 @@ _dispatch_event_loop_poke(dispatch_wlh_t wlh, uint64_t dq_state, uint32_t flags)
 			.ident  = 1,
 			.filter = EVFILT_USER,
 			.fflags = NOTE_TRIGGER,
-			.udata = (uintptr_t)DISPATCH_WLH_MANAGER,
+			.udata = (dispatch_kqueue_udata)DISPATCH_WLH_MANAGER,
```

Comment:
There seems to be a lot of these kind of changes here. It might make sense to break this out (`uintptr_t` -> `dispatch_kqueue_udata`) into its own change first, which can then be merged ahead of the rest of the changes.

Reply:
Yeah... I think kevent is a horrible mess with these changes, I hate it, but I don't see it could be made much better than this.

I think it would make sense to split into separate changes.